app/informativedialog.h

```diff
@@ -0,0 +1,23 @@
+#ifndef INFORMATIVEDIALOG_H
+#define INFORMATIVEDIALOG_H
+
+#include <QDialog>
+
+namespace Ui
+{
+class InformativeDialog;
+}
+
+class InformativeDialog : public QDialog
+{
+    Q_OBJECT
+
+public:
+    explicit InformativeDialog(QString title, QString description = QString(), QWidget *parent = 0 );
+    ~InformativeDialog();
+
+private:
+    Ui::InformativeDialog *ui;
+};
```

Comment:
Guidelines are helpers and instructions to stay within a certain design, they're not rules. They meant to be bend, twisted and broken in the most evil ways :P I know about Human interface guidelines and all that UI/UX stuff but I still don't agree with most of them. 

I'm not saying we should go the blender route, that would be extreme and maybe not even good, but a more personal UI should be possible even if it has to inherit OS-level consistency. I haven't looked too much into it yet but it's on my personal roadmap.